cypress/integration/login.js

```diff
@@ -1,47 +1,58 @@
-        cy.get('.user-info__title').contains('TestCon User 10')
-        cy.get('.main-nav').find('li').should('have.length', 2)
 
-        cy.get('.calendar__body').should('be.visible')
-        cy.get('.calendar--selected').contains(date.getDate())
+        loginVariables.getLoginUserDropdown().click({force:true})
+        loginVariables.getLoginUserName("TestCon User 10").click()
+        loginVariables.getLoginRoleIdList().click({force:true})
+        loginVariables.getLoginRoleName("Team Lead").click()
+        loginVariables.getSubmit().click()
+        timeLoggingVariables.getUrl().should('include', '/time-logging')
+        timeLoggingVariables.getPageTitle().contains('Timesheets')
+        calendarVariables.getCalendar().should('be.visible')
+        timeLoggingVariables.getTitleForm().should('be.visible')
+        timeLoggingVariables.getUserInfoTitle().contains('TestCon User 10')
+        timeLoggingVariables.getMainNavigationBar().should('have.length', 2)
+        calendarVariables.getCalendarBody().should('be.visible')
+        calendarVariables.getSelectedCalendarDay().contains(date.getDate())
     })
     it('Should verify each user role', function (){
         
-        let roles = ['User', 'Team Lead', 'Manager', 'Accountant', 'Admin']
-        let tabs = [1, 2, 5, 5, 6]
+        let roles = [
+            ['User',1],
+            ['Team Lead',2], 
+            ['Manager',5], 
+            ['Accountant',5], 
+            ['Admin',6]
+        ]
 
-        cy.visit('/')
-        roles.forEach (function(item, index) {
+        loginVariables.getRootPage()
+        for(let i = 0; i < roles.length; i++) {
             
-        cy.get('[id="loginForm.userId"]').click({force:true})
-        cy.get('[aria-label="TestCon User 10"]').click()
-        cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get("[aria-label="+'"'+item+'"'+"]").click()
-        cy.get('[type="submit"]').click()
-        cy.get('.main-nav').find('li').should('have.length', tabs[index])
-        cy.get('.main-header__user-info').click()
-        cy.get('.main-nav__link--active').contains('Time Logging').should('have.css', 'color', 'rgb(64, 76, 237)')
-        cy.get('.main-header__actions').contains('Log Out').click()
-    })
+            loginVariables.getLoginUserDropdown().click({force:true})
+            loginVariables.getLoginUserName("TestCon User 10").click()
+            loginVariables.getLoginRoleIdList().click({force:true})
+            loginVariables.getLoginRoleName(roles[i][0]).click()
+            loginVariables.getSubmit().click()
+            timeLoggingVariables.getMainNavigationBar().should('have.length', roles[i][1])
+            timeLoggingVariables.getMainHeaderUserName().click()
+            timeLoggingVariables.getActiveNavBarItem().contains('Time Logging').should('have.css', 'color', 'rgb(64, 76, 237)')
+            timeLoggingVariables.getMainHeaderActionList().contains('Log Out').click()
```

Comment:
In this context, [.contains('Log Out').] is used as not an assert, but a locator, so you might be better off creating 
timeLoggingVariables.getMainHeaderActionLogoutButton().click() 
timeLoggingVariables.getLogoutButton().click()